### ISSUE TITLE
Stale Nonce ? ----- From what i can make of it?

### ISSUE THREAD

**Issue by RUminingTwo** [ethereum-mining/ethminer#1074]
Hi, I have tried the latest build, by dropping in a replacing the older ethermine, with the 15.7dev, and when I start it, it starts loading the dag for a bit then errors out, and the cmd window quickly closes. What I can make out of the error, ( displays very quick and closes cmd window) is that its in yellow, and says " Stale nonce"" . This is on a 1060 6 gig card I am testing the latest release on.

If any help on this error, would be appreciated, Thank you

**Comment by AndreaLanfranchi** [comment]
Please report error and log. Also full cmd line used to start ethminer.
If window closes before you can see something edit your .bat file and add "pause" as last line.

**Comment by RUminingTwo** [comment]
Hi, Thank you for the quick reply....




setx GPU_FORCE_64BIT_PTR 0
setx GPU_MAX_HEAP_SIZE 100
setx GPU_USE_SYNC_OBJECTS 1
setx GPU_MAX_ALLOC_PERCENT 100
setx GPU_SINGLE_ALLOC_PERCENT 100

ethminer -U -S etc.pool.minergate.com:45777 -u ""myemail address"

pause


I ran the bat file, with the pause, and it paused. But the error doesn't show. When I hit the keyboard key to continue the cmd window closes with no error shown.

If I remove the pause. Its back again.......Stale Nonce with a bunch of numbers after the word Stale Nonce. about 10 digits long.

Not sure where the log file would be, as I have been running the previous version from a folder with  the following files in the folder.

cudart64_80.dll
ethermine.exe
ethermine.pdb
libcurl.dll
libmicrohttpd-dll.dll
msvcp140.dll
opencl.dll
vcdruntime140.dll

There is no log file in the folder, so I am not aware of where a log file would be created, or location outside of this folder with all the files.


Thank you

**Comment by AndreaLanfranchi** [comment]
It's related to #1069 
You have to be patient and wait for next release or step back to the release which was working for you.

**Comment by RUminingTwo** [comment]
Ok, I wasnt sure if it had anything to do with the expected version of cuda.dll's or the vc runtime lib files.

Not sure if it was me or what, but it does seem to load up quicker "prior to the error being generated".

I'll revert back to the prior build for now.

Thanks again

**Comment by AndreaLanfranchi** [comment]
Btw ... your bat file should be exactly this
(all other lines setx are not needed anymore as they are hardcoded into ethminer)

```
ethminer.exe -U -P stratum+tcp://<yourencodedemailaddress>@etc.pool.minergate.com:45777 
pause

```

If you do not know how to encode your email address simply use this
https://www.urlencoder.org/

**Comment by RUminingTwo** [comment]
Ah, Thank you, wasnt aware of the inclusion into the .exe now.

Nice, makes it much easier, and easier for wider adoption by end users

**Comment by RUminingTwo** [comment]
I had tried out """ ethminer.exe -U -P stratum+tcp://<yourencodedemailaddress>@etc.pool.minergate.com:45777 
pause"""""

with my email address included, coded or not, It didn't work for me?

The date of the working ethminer.exe is from january 15, so I am not sure which build ver it is..or if it is the right one.

**Comment by AndreaLanfranchi** [comment]
To know which version of ethminer you're on open a command prompt in the directory where ethminer is and type 

`ethminer.exe -V [Enter]`

(note that "V" has to be Uppercase)

**Comment by RUminingTwo** [comment]
Thank you
Version is, 13.0rc7r, it has been very stable but I now realize I am quite a bit behind on versions.

**Comment by RUminingTwo** [comment]
13.0rc7 not with a "r" at the end as my error above shows

**Comment by AndreaLanfranchi** [comment]
Solved in source releases 0.14 and master.
Binaries yet to be released.

**Comment by RUminingTwo** [comment]
Thank you ...
Best
